script/classification.js

```diff
@@ -0,0 +1,685 @@
+classification = {
```

Comment:
The generation code has a JSON.Parse but not this file. If not needed, then update the generation code.

Reply:
it has we are printing the out put and then we are wtiting it tot that file 	./tnf generate catalog javaScript > script/classification.js

Reply:
removed the pare on that file